files/Aboot/boot0.j2

```diff
@@ -494,6 +494,8 @@ write_platform_specific_cmdline() {
     if in_array "$sid" "Upperlake" "UpperlakeES" "UpperlakeElite"; then
         aboot_machine=arista_7060_cx32s
         flash_size=3700
+	cmdline_add memmap=512M!4G
+        cmdline_add mount_pmem=on
```

Comment:
@vdahiya12 don't need this new parameter, see below comment

Reply:
fixed